lib/octokit/client/milestones.rb

```diff
@@ -61,7 +61,7 @@ def create_milestone(repository, title, options = {})
       # @param number [String, Integer] ID of the milestone
       # @param options [Hash] A customizable set of options.
       # @option options [String] :title A unique title.
-      # @option options [String] :state (open) State: <tt>open</tt> or <tt>closed</tt>.
+      # @option options [String] :state (open) State: <tt>open</tt>, <tt>closed</tt>, or <tt>all</tt>.
```

Comment:
Same here, the `all` value works for listings, but isn't a valid value for specifying state on a milestone.